.github/workflows/build_and_test_clang_debug.yml

```diff
@@ -5,6 +5,8 @@ jobs:
   build:
     name: Build
     runs-on: ubuntu-18.04
+    outputs:
+      output1: ${{ steps.prepare_artifacts.outputs.test}}
```

Comment:
Please rename output1 for better readability

Reply:
renamed to err_output